build.js

```diff
@@ -397,6 +399,25 @@ function handle(options) {
 }
 
 function dataToHtml(skeleton, rawBrowsers, tests, compiler) {
+  var wrapWithScript = function(assertions) {
+    // wrap a single test in an array so we can deal with them consistently
+    if (assertions && !Array.isArray(assertions)) {
```

Comment:
If you’re checking isArray, you could just do `[assertions]`; concat is how you’d do it without the if check at all.

Reply:
Also, relying on Array.isArray won’t work on the pre-ES5 page.

Reply:
@ljharb it's a building tool and it already uses `Array.isArray`.

Reply:
ah, ok great - then scratch all my pre-ES5 comments :-) just the first one in this thread then.